src/Microsoft.IdentityModel.Tokens/Validation/Validators.TokenReplay.cs

```diff
@@ -60,16 +59,16 @@ public static partial class Validators
                     return new ValidationError(
                         new MessageDetail(
                             LogMessages.IDX10227,
-                            LogHelper.MarkAsUnsafeSecurityArtifact(securityToken, t => t.ToString())),
+                            securityToken),
                         ValidationFailureType.TokenReplayValidationFailed,
-                        typeof(SecurityTokenReplayDetectedException),
+                        typeof(SecurityTokenNoExpirationException),
```

Comment:
Is changing the exception type a breaking change?

Reply:
This code path is not yet public. The change simply aligns the new path with the existing one which returns a `SecurityTokenNoExpirationException` in this scenario.

Reply:
Thanks for clarifying! Resolving this comment